### ISSUE TITLE
Can not text Chinese because lack of Chinese input method when use keybord

### ISSUE THREAD

**Issue by xqqy** [ubports/ubuntu-touch#964]
- Device: nexus7 wifi(flo)
- Channel:develop
- Build:20181114-13629

### Steps to reproduce
1 connect a keybord

### Expected behavior
It should let me change my text input method with super+space just like what happened in other system.
### Actual behavior

I could only text English.
Even I chosed Chinese in text entry setting.

**Comment by dobey** [comment]
You need to enable the Chinese text entry for external keyboard in _System Settings_. When you connect an external keyboard, an additional option appears in the _Language & Text_ settings for it, and you can set the correct keyboard layout. Does setting the options in here resolve the problem for you? (I will agree that there needs to be some work done to improve integration of OSK and external keyboard text input options, though.)

**Comment by xqqy** [comment]
Yes I did set it, but opinion "Chinese"'s action is just as same as English.
In ibus and fcitx, the text input method is called "pinyin" and "wubi".

**Comment by xqqy** [comment]
Yes the virtual keyboard worked well. But when I connect my HARDWARE KEYBOARD via micro-usb otg, there is no such input mothods.

**Comment by UniversalSuperBox** [comment]
Oh, yes, sorry. The suggestions bar does not appear when an external keyboard is connected.

**Comment by dobey** [comment]
I've changed the title to better reflect the issue here, as this is not limited to Chinese language input methods.
